proposals/unified-show-diff.md

```diff
@@ -0,0 +1,134 @@
+# Title: Unified show/diff command for all output types
+
+- Enhancement Proposal PR: (leave this empty)
+- Contributors: dberenbaum
+
+# Summary
+
+Establish a unified command to combine and `show` or `diff` all outputs of
+specified types, including:
```

Comment:
I agree the different `show` subcommands are similar enough to get merged, and even more so for `diff` subcmds. but are `show`s similar enough to `diff`s, or do they have intrinsically different uses? In the latter case we would need to keep 2 top commands. BTW `exp show` is very different (experiments are not outputs).